cosmic/src/evolv2.f

```diff
@@ -2497,27 +2497,12 @@ SUBROUTINE evolv2(kstar,mass,tb,ecc,z,tphysf,
             dm1 = dm1*1.0d+03*mass(j1)/MAX(rad(j1),1.0d-04)
          endif
          kst = kstar(j2)
+
 *
-* Possibly mass transfer needs to be reduced if primary is rotating
-* faster than the orbit (not currently implemented).
-*
-*        spnfac = MIN(3.d0,MAX(ospin(j1)/oorb,1.d0))
-*        dm1 = dm1/spnfac**2
-*
-* Limit mass transfer to the thermal rate for remaining giant-like stars
-* and to the dynamical rate for all others.
+* Check if rad1 >= 10R1_Roche or if MS donor and qc > qcrit
+* Induce merger if true
 *
-         if(kstar(j1).ge.2.and.kstar(j1).le.9.and.kstar(j1).ne.7)then
-***
-* JH_temp ... this may be good for HG RLOF??
-*           if(kstar(j1).eq.2)then
-*              mew = rad(j1)/rol(j1) - 1.d0
-*              mew = 2.d0*mew
-*              dm1 = dm1*10.d0**mew
-*           endif
-***
-            dm1 = MIN(dm1,mass(j1)*tb/tkh(j1))
-         elseif(rad(j1).gt.10.d0*rol(j1).or.(kstar(j1).le.1.and.
+         if(rad(j1).gt.10.d0*rol(j1).or.(kstar(j1).le.1.and.
      &          kstar(j2).le.1.and.q(j1).gt.qc))then
```

Comment:
check that qc stuff is already taken care of by qcrits above

Reply:
confirming here that we use this to check to see if main sequence donor will merge with accretor.